README.md

```diff
@@ -1,5 +1,17 @@
-> Name to be found
-
-# PyFlag Bot
+# RootPythia
 
 **TODO: everything**
+
+### API Root-me
```

Comment:
Not related with this PR?

Reply:
No i think it was in my main branch :/

Reply:
Ok, let's keep it, it won't hurt anyone :shushing_face: